dotnet/samples/Concepts/Filtering/RetryWithFilters.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Net;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Connectors.OpenAI;
+
+namespace Filtering;
+
+/// <summary>
+/// This example shows how to perform retry with filter and switch to another model as a fallback.
+/// </summary>
+public class RetryWithFilters(ITestOutputHelper output) : BaseTest(output)
```

Comment:
We should also add a link to this from here:
https://github.com/microsoft/semantic-kernel/blob/main/dotnet/samples/Concepts/README.md

Reply:
Good catch, thank you! Added.